src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 6; i++) {
+            Ball ball1 = lottery.getRandomBall();
+            System.out.println(ball1);
+        }
         // create three balls using class Lottery and print information about them in console
```

Comment:
better remove comments

Reply:
made all but it is hard to read for me expecialy this 
    public Color getRandomColor() {
        return Color.values()[random.nextInt(Color.values().length)];
    }